.github/workflows/publish.yml

```
@@ -73,9 +77,16 @@ jobs:
 
       - name: publish
         shell: pwsh
+        env:
+          RELEASE_TAG: ${{ github.event.inputs.release_tag }}
         working-directory: src/D2L.Bmx
         run: |
-          dotnet publish -r $env:PLATFORM-x64 -o build
+          $version = "$env:RELEASE_TAG"
+          if ( -not $version -or $version -eq '' ){
+            $version = "v3.0.0"
+          }
+          $versionWithoutv = "$version".TrimStart("v")
+          dotnet publish /p:Version="$versionWithoutv" /p:InformationalVersion="$version" /p:IncludeSourceRevisionInInformationalVersion=false -r $env:PLATFORM-x64 -o build
```

nit: separate arguments in a long command into their own lines for readability
```suggestion
          dotnet publish `
            /p:Version="$versionWithoutv" `
            /p:InformationalVersion="$version" `
            /p:IncludeSourceRevisionInInformationalVersion=false `
            -r $env:PLATFORM-x64 `
            -o build
```